recipe/meta.yaml

```diff
@@ -36,22 +37,29 @@ outputs:
       ignore_run_exports_from:
         - python
         - numpy
+        - cross-python_{{ target_platform }}
       {% set mpi_prefix = "mpi_" + mpi %}
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ stdlib("c") }}
         - {{ compiler('cxx') }}
+        - cmake >=3.9
+        - make
+        - pkg-config
+        - {{ mpi }}  # [mpi == 'openmpi' and build_platform != target_platform]
+        # python needed for ufc discovery via ffc
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - numpy  # [build_platform != target_platform]
+        - fenics-ffc =={{ version }}  # [build_platform != target_platform]
       host:
         - libblas
         - libcblas
         - libboost-devel
-        - cmake >=3.9
-        - make
```

Comment:
these are build requirements, not host. Comes up in true cross compilation without emulation (i.e. osx-arm).